CHANGES.md

```diff
@@ -1,2 +1,11 @@
+Version 2.1:
+ - Change default Docker version to 1.11.2, was 1.5.0
```

Comment:
The internal update we're planning is to 1.11.2; we're holding off on 1.12 because 1.12 deprecates "tag -f", which we use everywhere. I figure it's better to have a default version than not - that makes it "just work" out of the box.